src/Sylius/Component/Resource/Model/ArchivableInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Resource\Model;
+
+/**
+ * @author Jan Góralski <jan.goralski@lakion.com>
+ */
+interface ArchivableInterface
```

Comment:
If you create `ArchivableInterface` in resource component then maybe you should move `ArchivableType` into `ResourceBundle` WDYT?